tests/quickchecks.rs

```diff
@@ -227,4 +234,32 @@ quickcheck! {
             assert_eq!(val, alloc);
         }
     }
+
+    fn all_allocations_in_a_chunk(values: Vec<BigValue>) -> () {
+        let b = Bump::new();
+        let allocated: Vec<&BigValue> = values.into_iter().map(|val| b.alloc(val) as &_).collect();
+        let chunks: Vec<(*mut u8, usize)> = unsafe { b.iter_allocated_chunks_raw() }.collect();
+        for alloc in allocated.into_iter() {
+            assert!(chunks.iter().any(|&(ptr, size)| {
+                let ptr = ptr as usize;
+                let chunk = (ptr, ptr + size);
+                contains(chunk, range(alloc))
+            }));
+        }
+    }
+
+    fn chunks_and_raw_chunks_are_same(values: Vec<BigValue>) -> () {
+        let mut b = Bump::new();
+        for val in values {
+            b.alloc(val);
+        }
+        let raw_chunks: Vec<(_, _)> = unsafe { b.iter_allocated_chunks_raw() }.collect();
+        let chunks: Vec<&[_]> = b.iter_allocated_chunks().collect();
+        for ((ptr, size), chunk) in raw_chunks.into_iter().zip(chunks) {
+            if let Some(byte) = chunk.get(0) {
```

Comment:
If we want to check the address of empty chunks, seems like there isn't a technically correct way to do it in stable Rust, [this nightly method](https://doc.rust-lang.org/stable/std/mem/union.MaybeUninit.html#method.slice_as_ptr) is what we need.

In practice, `chunk.as_ptr() as *const u8` should work (using the slice's `.as_ptr()` instead of `MaybeUninit`, assuming that the address of the `MaybeUninit` is the same as the address of the contained value) - which is true for `std`'s current implementation and "should" always be, but AFAICT not actually guaranteed. Maybe that's an acceptable amount of unspecified behavior (not actual UB) to rely on in a test case?